ui/store/actions.js

```diff
@@ -1657,18 +1657,32 @@ export function importTokens(tokensToImport) {
 }
 
 /**
- * To add ignored tokens to state
+ * To add ignored token addresses to state
  *
- * @param tokensToIgnore
+ * @param options
+ * @param options.tokensToIgnore
+ * @param options.dontShowLoadingIndicator
  */
-export function ignoreTokens(tokensToIgnore) {
+export function ignoreTokens({
+  tokensToIgnore,
+  dontShowLoadingIndicator = false,
+}) {
+  const _tokensToIgnore = Array.isArray(tokensToIgnore)
```

Comment:
There are many places where `ignoreTokens` has replaced `removeAndIgnoreToken` (which is removed as part of this PR). `TokensController.ignoreTokens()` expects an array of `tokensToIgnore` where `removeAndIgnoreToken` expected a single tokenAddress. I've adapted this action function to accept either an array or a single tokenAddress (and wrap the address in an array before passing along to `TokensController`) rather than wrap single tokenAddresses in an array everywhere we call the action function.

Reply:
Unless this is a pattern we repeat, we should probably just use the array syntax everywhere.